packages/mobile/src/components/BottomSheet.tsx

```diff
@@ -0,0 +1,141 @@
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import { Spacing } from '@celo/react-components/styles/styles'
+import React, { useState } from 'react'
+import { Dimensions, LayoutChangeEvent, StyleSheet, View } from 'react-native'
+import { TouchableWithoutFeedback } from 'react-native-gesture-handler'
+import Animated, { useAnimatedStyle, useSharedValue } from 'react-native-reanimated'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+import { useShowOrHideAnimation } from 'src/components/useShowOrHideAnimation'
+
+interface Props {
+  isVisible: boolean
+  onBackgroundPress?: () => void
+  children?: JSX.Element
+}
+
+const MIN_EMPTY_SPACE = 100
+
+function BottomSheet({ children, isVisible, onBackgroundPress }: Props) {
```

Comment:
Mainly I copy pasted this class from `TokenBottomSheet`.
I will create a next PR with a refactor to start using only this class instead of having both.